tutorials/introductory/usage.py

```diff
@@ -413,27 +412,42 @@ def my_plotter(ax, data1, data2, param_dict):
 # ============   ================================================================
 # Backend        Description
 # ============   ================================================================
-# GTKAgg         Agg rendering to a :term:`GTK` 2.x canvas (requires PyGTK_ and
-#                pycairo_ or cairocffi_; Python2 only)
+# Qt5Agg         Agg rendering in a :term:`Qt5` canvas (requires PyQt5_).  This
+#                backend can be activated in IPython with ``%matplotlib qt5``.
+# ipympl         Agg rendering embedded in a Jupyter widget.  This can be enabled
```

Comment:
? This doesn't come with Matplotlib; should there be some reference for installing it (like the other requirements)?

Reply:
Yes, PR shortly.

Reply:
xref #9126